.github/workflows/conformance.yml

```diff
@@ -0,0 +1,79 @@
+name: Manually run upstream conformance test
+
+on:
+  workflow_dispatch:
+    inputs:
+      antrea-version:
+        description: The Antrea version to test. It could be a SHA-1 value, a branch, or a tag (e.g. a7b012b, release-1.12, v1.12.0).
+        default: main
+        required: true
+      k8s-version:
+        description: K8s version (e.g. v1.27.1). Kind's default K8s version will be used if empty.
```

Comment:
Can we log somewhere if the default K8s is used?

Reply:
The build log and sonobuoy log will have the K8s version info.